web/frontend/src/components/navigation/components/scene.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang='ts'>
+
+import * as THREE from 'three';
+import { T, useThrelte, useRender, type CurrentWritable } from '@threlte/core';
+import { type Map } from 'maplibre-gl';
+import { obstacles, zoomLevels } from '../stores';
+import { createCameraTransform } from '../utils';
+import type { Mat4, Obstacle } from '../types';
+
+export let map: Map;
+export let viewProjectionMatrix: CurrentWritable<Float32Array | Mat4>;
+
+const { renderer, scene, camera } = useThrelte();
+
+renderer!.autoClear = false;
+
+// This clips against the map so that interesecting objects will not render over the map
```

Comment:
[ultra nit] spelling error "interesecting" should "intersecting"

Reply:
Fixed!